src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1606,9 +1605,12 @@ public void validateDeleteRequest(Dataset dataset, List<DataDeletionTableModel>
         final FieldList columns = result.getSchema().getFields();
         final List<Map<String, Object>> values = new ArrayList<>();
         result.iterateAll().forEach(rows -> {
-            final var rowData = columns.stream().map(Field::getName)
-                .collect(Collectors.toMap(Function.identity(),
-                    columnName -> rows.get(columnName).getValue()));
+            final Map<String, Object> rowData = new HashMap<>();
```

Comment:
I'd like to understand what's wrong with the stream form at some point, but it's better to get it working for now.